department-of-veterans-affairs/vets-website
# VAGOV-4688: Facility | Update "Get updates from us" component

## Description
As a user on the VA Pittsburgh site, I want to be able to sign up for news and announcements, emergency notifications, and follow social media accounts so I can stay updated on VA Pittsburgh goings on.

## Testing done
locally with local && dev 
1. build site
2. browse to http://localhost:3001/pittsburgh-health-care
3. scroll to the bottom to check out the "Get updates from us" component
4. make sure the links show properly according to mock ups: https://app.zeplin.io/project/5d014fd159c9931608872c98/screen/5d24c63271765a98d1fb9b4f
5. check these other urls for the component to show up, look correct, and link to the correct place
http://localhost:3001/pittsburgh-health-care/

http://localhost:3001/pittsburgh-health-care/contact-us/

http://localhost:3001/pittsburgh-health-care/locations/hj-heinz/

http://localhost:3001/pittsburgh-health-care/locations/pittsburgh-va-medical-center-university-drive/

http://localhost:3001/pittsburgh-health-care/locations/beaver-county-outpatient-clinic/

http://localhost:3001/pittsburgh-health-care/locations/belmont-county-outpatient-clinic/

http://localhost:3001/pittsburgh-health-care/locations/fayette-county-outpatient-clinic/

http://localhost:3001/pittsburgh-health-care/locations/washington-county-outpatient-clinic/

http://localhost:3001/pittsburgh-health-care/locations/westmoreland-county-outpatient-clinic/

## Screenshots
![localhost_3001_pittsburgh-health-care_locations_pittsburgh-va-medical-center-university-drive_](https://user-images.githubusercontent.com/19178435/61157114-08c47880-a4aa-11e9-8682-1e18c5d86b33.png)


## Acceptance criteria


## Definition of done
